src/pytest_inmanta_lsm/remote_orchestrator.py

```diff
@@ -45,6 +45,9 @@ def __init__(
         settings: Dict[str, Union[bool, str, int]],
         noclean: bool,
         ssh_port: str = "22",
+        ssl: str = "false",
```

Comment:
I think it would be good to use the type `bool` here. That way the type checker will enforce that we don't pass anything else but `"True"` or `"False"` in this method and also into the `inmanta_config.Config.set()` method.